src/source-control/GitSourceControl.ts

```diff
@@ -44,4 +44,8 @@ export class GitSourceControl implements SourceControl {
   async attachSttackBranchesToCommits(commits: Commit[]): Promise<void> {
     // TODO: Implement
   }
+
+  async createSttackBranchForCommit(branch: BranchName, commit: Commit): Promise<void> { 
```

Comment:
Looks like this is intended to replace `attachSttackBranchesToCommits`? I kinda prefer the `attach` name hahaha

Reply:
I wasn't sure if they were doing the same thing hahaha